src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/bar.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Bar;
+
+class BarTagClass
+{
+    private $bar;
+
+    public function __construct($bar = '')
```

Comment:
that's a lot of dead code for a fixture, please reduce them to empty classes if possible.

Reply:
I'll clean up that classes.